build/configs/artik053/audio/defconfig

```diff
@@ -423,9 +423,10 @@ CONFIG_AUDIO_ALC5658=y
 CONFIG_ALC5658_INITVOLUME=250
 CONFIG_ALC5658_INFLIGHT=2
 CONFIG_ALC5658_MSG_PRIO=1
-CONFIG_ALC5658_BUFFER_SIZE=8192
-CONFIG_ALC5658_NUM_BUFFERS=4
+CONFIG_ALC5658_BUFFER_SIZE=6400
+CONFIG_ALC5658_NUM_BUFFERS=2
 CONFIG_ALC5658_WORKER_STACKSIZE=768
+CONFIG_ALC5658_I2S_TIMEOUT=100
```

Comment:
This is not the i2s timeout. We are actually expecting to receive one i2s dma interrupt every 100 ms. So, please change this config to I2S_IRQ_DURATION. Time out has to be calculated differently as mentioned in below comments.

Reply:
that value used for buffer container's time out value.
If we change it to 500ms, we must use huge buffer.